### ISSUE TITLE
Cannot resolve action plugin 

### ISSUE THREAD

**Issue by maxnasonov** [ansible/ansible-lint#3365]
##### Summary

A locally defined (in the root of the Ansible project) action cannot be resolved: 

```shell
$ ansible-lint
WARNING  Listing 1 violation(s) that are fatal
syntax-check[specific]: couldn't resolve module/action 'azure_storage_blob'. This often indicates a misspelling, missing collection, or incorrect module path.
roles/eventstoredb/tasks/main.yml:34:7


                  Rule Violation Summary                   
 count tag                    profile rule associated tags 
     1 syntax-check[specific] min     core, unskippable    

Failed after : 1 failure(s), 0 warning(s) on 283 files.

```

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
$ ansible-lint --version
ansible-lint 6.15.0 using ansible 2.14.5
```

- ansible installation method: pip
- ansible-lint installation method: pip
- os: macOS 12.6.5

##### STEPS TO REPRODUCE

```console (paste below)
$ tree
action_plugins
├── azure_storage_blob.py
library/
└── azure_storage_blob.py
roles/eventstoredb/
...

$ cat library/azure_storage_blob.py
# Implemented via the action plugin ../action_plugins/azure_storage_blob.py

$ cat roles/eventstoredb/tasks/main.yaml
...
- name: Download
  azure_storage_blob:
    url: ...
...
```

##### Desired Behavior

ansible-lint should process the task without failures.

##### Actual Behavior

See above.

**Comment by cidrblock** [comment]
Can you try adding an `ansible.cfg` file to root of the workspace with the following?

```
[defaults]
library=.
```

This should help ansible find these plugins.

(just a work around for now, the underlying issues related to project layout are a bigger issue being worked on)

**Comment by ajinkyau** [comment]
@ssbarnea I've tried to validate this per @cidrblock's comment and it did work for me.
Also, if I don't use an `ansible.cfg` but include this role in a playbook, validate with `ansible-lint` command and also `ansible-playbook --syntax-check` I don't get any errors.

When the playbook is executed it gives desired results with no errors.

**Comment by maxnasonov** [comment]
For the sake of completion, I've finally been able to verify the workaround proposed by @cidrblock - it resolved the original issue for me as well. Thanks!
